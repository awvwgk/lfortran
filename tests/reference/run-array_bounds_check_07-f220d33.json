{
    "basename": "run-array_bounds_check_07-f220d33",
    "cmd": "lfortran --no-color {infile}",
    "infile": "tests/errors/array_bounds_check_07.f90",
    "infile_hash": "e41af0e94a994140f23d1bec748d4d21d0762f8500b3fa0ef54e18ca",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "run-array_bounds_check_07-f220d33.stderr",
    "stderr_hash": "8ca1b422f13d8baddd3ebd56f367732c1cf88d12b0ff160c594ca991",
    "returncode": 1
}