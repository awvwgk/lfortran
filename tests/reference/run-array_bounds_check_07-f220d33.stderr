At 12:9 of file tests/errors/array_bounds_check_07.f90
Runtime Error: Size mismatch in binary operation with operands 'x' and 'y'

Size of 'x' is 3 and size of 'y' is 4
